### ISSUE TITLE
As a Founder, I want a menu option on the left sidebar to view all the Connections for the Type Founders

### ISSUE THREAD

**Issue by zachpritchard** [AlchemistAcc/portal#13]
The founder should be able to click the Founders sub option under Connections in the left side bar and view the recommended founders associated with the Weekly Focus. Here's a wireframe of the layout:
![image](https://cloud.githubusercontent.com/assets/12280257/7739914/3314a01e-ff1e-11e4-8b51-c71bc106a83e.png)

The Send Email button will allow the Founder to write a message and send it to the listed founder's email address on file. I would like to give this the ability to send an email directly from our platform for measurement purposes. If this isn't possible, the button can simply open a new message in their email client.

Also, I will provide more detail for the order based on Weekly Focus when we start layering in the Recommendation Logic. Thanks!

**Comment by schrizsz** [comment]
- founders card - put exactly the fields on screenshot (use ellipsis if company name is long , also for other fields), show only couple of tags, all cards the same size.
- (x) on card should record based on the current weekly focus (table unwanted_recommendations cols: focus, user_id) 
- at the moment just show the founders

**Comment by schrizsz** [comment]
@zachpritchard should we create separate issue for the right top x functionality, it is kind of general and should depend on #22 ?

**Comment by zachpritchard** [comment]
@schrizsz That's definitely a good idea since it does have some logic behind it. I'll create a new story for it then. Thanks!

**Comment by schrizsz** [comment]
-send email should open in new tab (target=_blank) 
-the page should have title as in mockup screenshot

**Comment by schrizsz** [comment]
From the meeting: @VeikoSoomets 
Title, User name are incorrectly capitalised. It should preserve how it's input.
Title, Company name (without prefix) (Please use Alchemist company name as a link if founder) should be on the same line.
The buttons should be centered relative to the whole card
Also make sure Users name , Company name are cut nicely and the card does not overflow (use css ellipsis?)
Also the left side menu is not responsive please see the theme I was using initially (it had responsive menu I guess, but it had some bugs) http://startbootstrap.com/template-overviews/sb-admin-2/ http://ironsummitmedia.github.io/startbootstrap-sb-admin-2/pages/index.html
